### ISSUE TITLE
Feature request: reformat jsdoc blocks

### ISSUE THREAD

**Issue by jamietre** [joelday/vscode-docthis#58]
VS Code doesn't seem to do anything in the way of formatting jsdoc text when it renders intellisense, so if you don't happen to break lines in your descriptions at the right place (80?) it looks ugly. Would be nice to have a feature to automatically reformat text blocks in jsdoc comments to the correct width.

**Comment by jamietre** [comment]
Since I posted this - VS Code seems to auto format. There must have been a change related to this.

However I think the feature is still useful (even if not necessary for VS code intellisense to look good). Having a keyboard shortcut to reformat a jsdoc block using some basic user configurable settings would be awesome for those of us who are anal about how our code looks. 

Whenever you edit a substantial method description, you end up doing a fair amount of manual reformatting, or just accept ugliness. I hate accepting ugliness so I have to do work that I feel like a computer should do for me :)

I saw your request for assistance. I'd love to help with this project (I love it) though I have limited experience with the VS Code extension API. The actual logic for this is well within my ability tho.
